content_scripts/storageLoader.js

```diff
@@ -20,6 +26,14 @@ function addDarkMode() {
     link.type = "text/css";
     link.rel = "stylesheet";
     document.getElementsByTagName("head")[0].appendChild(link);
+    let moodleLogoImg = document.querySelector(MOODLE_LOGO_SELECTOR)
+    let hujiLogoImg = document.querySelector(HUJI_LOGO_SELECTOR)
+
+    orgMoodleLogoSrc = moodleLogoImg.getAttribute("src");
+    orgHujiLogoSrc = hujiLogoImg.getAttribute("src");
+
+    moodleLogoImg.setAttribute("src","https://i.ibb.co/t3CTQ0t/moodle-logo-darkmode-withlogo.png")
+    hujiLogoImg.setAttribute("src","https://i.ibb.co/6mdGRfv/moodle-huji-logo-darkmode.png")
```

Comment:
You can add the images to our `/images` folder, it will be safer because I don't know how long does "ibb" will save the images for.